packages/manager/apps/catalog/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useState, useEffect } from 'react';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsLink } from '@ovhcloud/ods-components/link/react';
+import { useLocation } from 'react-router-dom';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import FilterItem from './FilterItem';
+import LoadingFilterItem from './LoadingFilterItem';
```

Comment:
I think we can group the imports for better readability.

1. First react & its library related imports
2. ODS related imports
3. Super-components imports
4. project related imports
```suggestion
import React, { useState, useEffect } from 'react';
import { useLocation } from 'react-router-dom';
import { useTranslation } from 'react-i18next';

import { OsdsButton } from '@ovhcloud/ods-components/button/react';
import { OsdsText } from '@ovhcloud/ods-components/text/react';
import { OsdsLink } from '@ovhcloud/ods-components/link/react';
import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';

import FilterItem from './FilterItem';
import LoadingFilterItem from './LoadingFilterItem';
```

Reply:
👍